packages/services/hmi-server/docker/jvm/Dockerfile.jvm

```diff
@@ -1,7 +1,7 @@
 ####
 # This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
 ###
-FROM registry.access.redhat.com/ubi8/openjdk-17:1.14-10
+FROM registry.access.redhat.com/ubi8/openjdk-17:1.14-10.1675788279
```

Comment:
Are we sure we want to do that.
I wonder if your branch is out of date with `main`?